scripts/prompt_processing_summary.py

```diff
@@ -44,38 +63,98 @@ def make_summary_message(day_obs):
+            sfm_counts + dia_counts, sfm_outputs, sfm_counts + dia_counts - sfm_outputs
+        )
+    )
 
-    dia_visit_detector = set([(x.dataId['visit'], x.dataId['detector']) for x in b.registry.queryDatasets("apdb_marker")])
-    output_lines.append("Number of successful DIA attempted: {:d}".format(len(dia_visit_detector)))
+    dia_visit_detector = set(
+        [
+            (x.dataId["visit"], x.dataId["detector"])
+            for x in b.registry.queryDatasets("apdb_marker")
+        ]
+    )
+    output_lines.append(
+        "- ApPipe: {:d} attempts, {:d} succeeded, {:d} failed.".format(
+            dia_counts, len(dia_visit_detector), dia_counts - len(dia_visit_detector)
+        )
+    )
 
-    missing_dias = set(log_visit_detector - dia_visit_detector)
-    missing_visits = [x[0] for x in missing_dias]
-    output_lines.append("Number of unsuccessful DIA attempts (no resulting apdb_marker): {:d}".format(len(missing_dias)))
+    output_lines.append(
+        f"<https://usdf-rsp-dev.slac.stanford.edu/times-square/github/lsst-dm/vv-team-notebooks/PREOPS-prompt-error-msgs?day_obs={day_obs}&instrument=LATISS&ts_hide_code=1|Full Error Log>"
+    )
 
-    output_lines.append(f"<https://usdf-rsp-dev.slac.stanford.edu/times-square/github/lsst-dm/vv-team-notebooks/PREOPS-prompt-error-msgs?day_obs={day_obs}&instrument=LATISS&ts_hide_code=1|Full Error Log>")
+    raws = {r.id: r.group for r in raw_exposures}
+    log_group_detector = {
+        (raws[visit], detector) for visit, detector in log_visit_detector
+    }
+    df = get_status_code_from_loki(day_obs)
+    df = df[(df["instrument"] == "LATISS") & (df["group"].isin(raws.values()))]
+
+    status_groups = df.set_index(["group", "detector"]).groupby("code").groups
+    for code in status_groups:
+        counts = len(status_groups[code])
+        output_lines.append(f"- {counts} counts have status code {code}.")
+
+        indices = status_groups[code].intersection(log_group_detector)
+        if not indices.empty and code != 200:
+            output_lines.append(f"  - {len(indices)} have outputs.")
+            counts -= len(indices)
+
+        match code:
+            case 500:
+                df = get_timeout_from_loki(day_obs)
+                df = df[
+                    (df["instrument"] == "LATISS") & (df["group"].isin(raws.values()))
+                ].set_index(["group", "detector"])
+                indices = status_groups[code].intersection(df.index)
+                if not indices.empty:
+                    output_lines.append(f"  - {len(indices)} timed out.")
+                    counts -= len(indices)
+                if counts > 0:
+                    output_lines.append(f"  - {counts} to be investigated.")
+
+    output_lines.append(
+        f"<https://usdf-rsp-dev.slac.stanford.edu/times-square/github/lsst-sqre/times-square-usdf/prompt-processing/groups?date={day_obs}&instrument=LATISS&survey={survey}&mode=DEBUG&ts_hide_code=1|Timing plots>"
```

Comment:
What does `&mode=DEBUG` do?

Reply:
In that Times Square notebook, written before we started making this reporting tool, I wanted a "report" mode versus a "debug" mode. The former is brief and meant for people who don't care about PP internals.  The latter has a lot more details which the PP team can use to know what might have gone wrong.  Now we have this reporting tool, I probably can go back and think again whether it makes sense.